lib/resty/core/shdict.lua

```diff
@@ -387,6 +403,40 @@ local function shdict_flush_all(zone)
 end
 
 
+local function shdict_get_pool_slot_count(zone)
+    zone = check_zone(zone)
+
+    C.ngx_http_lua_ffi_shdict_get_stats_slot_count(zone, slot_count_buf)
+    return tonumber(slot_count_buf[0])
+end
+
+
+local function shdict_get_stats(zone, slots)
+    zone = check_zone(zone)
+
+    C.ngx_http_lua_ffi_shdict_get_stats_slot_count(zone, slot_count_buf)
```

Comment:
We discard the return value here? Better do error handling properly?

Reply:
Currently `C.ngx_http_lua_ffi_shdict_get_stats_slot_count` and `C.ngx_http_lua_ffi_shdict_get_stats_slots_and_min_size` always return `NGX_OK`, so I omit the error handling?
https://github.com/openresty/lua-nginx-module/pull/1149/files

Should I do more defensive check in above two C APIs and do error handling in Lua side?

Reply:
@hnakamur It's better for the Lua wrapper not to assume the internal workings of the C API. If the return value is never meaningful we can simply make the C API void typed.